src/qibolab/platforms/platform.py

```diff
@@ -30,14 +30,37 @@ def disconnect(self):
         self.design.disconnect()
         self.is_connected = False
 
-    def execute_pulse_sequence(self, sequence, nshots=1024, relaxation_time=None):
+    def execute_pulse_sequence(
+        self, sequence, nshots=1024, relaxation_time=None, fast_reset=False, sim_time=10e-6, acquisition_type=None
+    ):
         if relaxation_time is None:
             relaxation_time = self.relaxation_time
-        return self.design.play(self.qubits, sequence, nshots=nshots, relaxation_time=relaxation_time)
+        if fast_reset is True:
+            fast_reset = {}
+            for qubit in self.qubits.values():
+                if "c" in str(qubit.name):
+                    pass
+                else:
+                    fast_reset[qubit.name] = self.create_RX_pulse(qubit=qubit.name, start=0)
```

Comment:
```suggestion
                if "c" not in str(qubit.name):
                    fast_reset[qubit.name] = self.create_RX_pulse(qubit=qubit.name, start=0)
```

Reply:
And explain somewhere what does it mean that check.

A nice option might be to split the check into a dedicated function:
```py
def my_check_on_qubit(qubit: Qubit) -> bool:
	"""Check explanation."
    return "c" not in str(qubit.name)
```

It is not a bad thing to write small functions: you can reuse, document in the docstring, and a good function name is already self-documenting, if telling enough.

Reply:
It's temporal way of checking if a qubit is a coupler or not. We didn't have a Qubit.iscoupler or anything so I just added a c to the name of the coupler qubits. I can try to add this option to the Qubit class.

Reply:
That small function would be really useful I added Qubit.flux_coupler so it would be something like this: